entrypoint.sh

```diff
@@ -55,6 +55,11 @@ if [ -n "$VAULT_ADDR" ]; then
     export VAULT_TOKEN
   fi
 
+  if [ -z "$VAULT_TOKEN" ] && [ -n "$VAULT_AUTH_ROLE" ]; then
+    VAULT_TOKEN=$(vault login -token-only -method="${VAULT_AUTH_PROVIDER:-aws}" role="$VAULT_AUTH_ROLE")
```

Comment:
The parameter `role=` is not valid for all auth providers, so this will fail to login even if that auth provider is configured in vault if it doesn't accept `role` as a parameter.

Reply:
removed as I was able to merge with the ECS path